bundle.yaml.j2

```diff
@@ -74,5 +73,7 @@ relations:
   - [identity-platform-login-ui-operator:ui-endpoint-info, hydra:ui-endpoint-info]
   - [identity-platform-login-ui-operator:ui-endpoint-info, kratos:ui-endpoint-info]
   - [identity-platform-login-ui-operator:kratos-endpoint-info, kratos:kratos-endpoint-info]
-  - [traefik-admin:certificates, tls-certificates-operator:certificates]
-  - [traefik-public:certificates, tls-certificates-operator:certificates]
+  - [traefik-admin:certificates, self-signed-certificates:certificates]
+  - [traefik-public:certificates, self-signed-certificates:certificates]
+  - [traefik-admin:receive-ca-cert, self-signed-certificates:send-ca-cert]
+  - [traefik-public:receive-ca-cert, self-signed-certificates:send-ca-cert]
```

Comment:
Question: Why does traefik need the ca cert? Is this needed for traefik to verify in-cluster HTTPS communication (e.g. if Hydra is serving through https)?

Reply:
It's most likely not needed, I removed these relations. (I run the tests and their positive without this relation.)